File: modules/coupon-domain/src/main/java/com/huythanh0x/udemycoupons/dto/ReviewResponseDTO.java
package com.huythanh0x.udemycoupons.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * DTO for instructor response to a review.
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ReviewResponseDTO {
    private String content;
    private String contentHtml;
    private String created;
    private String createdFormatted; // e.g., "18 hours ago"
    private ReviewUserDTO user;
}

File: modules/coupon-domain/src/main/kotlin/com/thanh0x/coursedeal/dto/ReviewResponseDTO.kt
package com.thanh0x.coursedeal.dto

/**
 * DTO for instructor response to a review.
 */
data class ReviewResponseDTO(
    val content: String? = null,
    val contentHtml: String? = null,
    val created: String? = null,
    // e.g., "18 hours ago"
    val createdFormatted: String? = null,
    val user: ReviewUserDTO? = null,
)
